docs/configuration/arguments.md

```diff
@@ -278,7 +280,7 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
 - `distributor.ha-tracker.failover-timeout`
    If we don't receive any samples from the accepted replica for a cluster in this amount of time we will failover to the next replica we receive a sample from. This value must be greater than the update timeout (default 30s)
 - `distributor.ha-tracker.store`
-   Backend storage to use for the ring (consul, etcd, inmemory). (default "consul")
+   Backend storage to use for the ring (consul, etcd, inmemory). Inmemory only works if there is a single distributor. (default "consul")
```

Comment:
_Same comment here_

Reply:
I removed the comment about the inmemory and added multi.